sklearn/metrics/_regression.py

```diff
@@ -286,10 +286,12 @@ def mean_pinball_loss(
 def mean_absolute_percentage_error(
     y_true, y_pred, sample_weight=None, multioutput="uniform_average"
 ):
-    """Mean absolute percentage error regression loss.
+    """Mean absolute percentage error (MAPE) regression loss.
 
-    Note here that we do not represent the output as a percentage in range
-    [0, 100]. Instead, we represent it in range [0, 1/eps]. Read more in the
+    Note here that the output is not a percentage in range [0, 100].
```

Comment:
Maybe, we should also state the we use no unit, i.e. 100 is not 100 % but really 1e2.

Reply:
Agreed, all the more that the [Wikipedia](https://en.wikipedia.org/wiki/Mean_absolute_percentage_error) definition (and the keras implementation, for instance) use a 100 multiplier. I guess we could only say `Note here that the output is not a percentage` so that it is clear that we do not multiply by 100.